docs/cmdline-opts/ca-native.md

```diff
@@ -12,7 +12,7 @@ See-also:
   - capath
   - insecure
 Example:
-  - --ca-native $URL
+  - --ca-native
```

Comment:
Why remove the URL? Without it, the example is incomplete

Reply:
Yep, just realized this, it's an example. Thanks for noticing!